README.md

```diff
@@ -1,6 +1,20 @@
-ToDoList
-========
+# An application to make Todo list
 
-Base du projet #8 : Améliorez un projet existant
+## Installation
 
-https://openclassrooms.com/projects/ameliorer-un-projet-existant-1
+---
+
+1.  Clone this repository.
+
+2.  With a CLI, place you in the project folder and run "composer install" command.
+
+3.  In .env, choose your SGBD by remove the # before the DATABASE_URL required and complete asked informations. Then, type in the CLI the 2 following commands :
```

Comment:
Keep the language consistent, "SGBD" is a French acronym ^^

```suggestion
3.  In .env, choose your DBMS by remove the # before the DATABASE_URL required and complete asked informations. Then, type in the CLI the 2 following commands :
```

Reply:
Yes, of course, sorry